manifests/u/Upscayl/Upscayl/2.7.5/Upscayl.Upscayl.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Upscayl.Upscayl
+PackageVersion: 2.7.5
+InstallerType: nullsoft
+UpgradeBehavior: install
+ReleaseDate: 2023-09-01
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/upscayl/upscayl/releases/download/v2.7.5/upscayl-2.7.5-win.exe
+  InstallerSha256: 34C54EAD3884F16F5F48D50BDC7D65D89D3D2FD3E71BC6CC08F3F68093FF637E
+  AppsAndFeaturesEntries:
+    - DisplayName: Upscayl 2.7.5
+      DisplayVersion: 2.7.5

```

Comment:
```suggestion
```
DisplayVersion should not be used when it's equal to PackageVersion. See https://github.com/microsoft/winget-pkgs/pull/65816#issue-1301301147